cfpb/cfgov-refresh
# Mortgage Performance: minor bugs around delinquency category colors and misworded comments

This PR makes two minor fixes to Mortgage Performance Trends.
 - The first makes sure that line comparison charts take the assigned color of their delinquency category. In other words, 90 day delinquencies should use the Navy color rather than Pacific Blue.
- The second corrects some miswordings in two function comment descriptions to be more accurate of what's actually in the functions. (Discovered when looking at another bug.)

## Changes

- Ensure that line comparison charts from different delinquency categories use different colors
- Fix two mistakenly worded comments to properly refer to states with non-metro areas and states with valid counties.

## Testing

1. Pull down this branch and `./setup.sh`
2. Set up the [mortgage performance API](https://github.com/cfpb/cfgov-refresh/pull/3189).
3. Create a [mortgage chart block, mortgage map block, and page](https://github.com/cfpb/cfgov-refresh/pull/3260) for 90-day delinquencies.
4. View the page. Both the chart and the map should use the color Navy when displaying the data.

## Screenshots

### Current status on production

![screen shot 2017-10-31 at 3 29 36 pm](https://user-images.githubusercontent.com/1183435/32245080-ff021280-be50-11e7-89bd-af0d11d8f725.png)

### Expected status after this PR

![screen shot 2017-10-31 at 3 29 45 pm](https://user-images.githubusercontent.com/1183435/32245089-03ebf248-be51-11e7-8bad-2dca4d5c9c50.png)

## Checklist
